pkg/validation/component.go

```diff
@@ -164,6 +164,7 @@ func (c *component) UpdateUser(ctx context.Context, userID string, user api.User
 
 		var userDB = dto.DBUser{
 			UserID:           &uID,
+			BirthLocation:    user.BirthLocation,
```

Comment:
Why this is added ?

Reply:
Because it was missing. The struct contains the birthlocation but the code was missing it. 
If there is a reason we should exclude it, let me know @harture .

Reply:
ok